app/src/main/res/xml/search_widget_info.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:initialKeyguardLayout="@layout/search_widget"
+    android:initialLayout="@layout/search_widget"
+    android:minHeight="40dp"
+    android:minWidth="180dp"
+    android:previewImage="@drawable/search_appwidget_preview"
+    android:resizeMode="horizontal|vertical"
+    android:updatePeriodMillis="86400000"
+    android:widgetCategory="home_screen"></appwidget-provider>
```

Comment:
You could simply use a closing brace, `/>`, and remove the closing tag, `</appwidget-provider>`.